src/libs/antares/study/parts/wind/series.cpp

```diff
@@ -72,6 +72,16 @@ int DataSeriesWindSaveToFolder(DataSeriesWind* d, const AreaName& areaID, const
     return ret;
 }
 
+double* DataSeriesWind::getAvailablePower(unsigned int year) const
+{
+    return timeSeries[getIndex(year)];
+}
+
+unsigned int DataSeriesWind::getIndex(unsigned int year) const
+{
+    return (timeSeries.width != 1) ? (long)timeseriesNumbers[0][year] : 0;
+}
+
```

Comment:
We start to have some code duplication with **clusters** (thermal or renewable) : classes for time series look very much the same : same interface, and even same body.
Furthermore, we expect more code duplication to come : when changes will reach load, solar, ...
We should think about a way to gather what's common in a single class.